bundles/AdminBundle/src/HttpFoundation/JsonResponse.php

```diff
@@ -28,7 +29,7 @@ class JsonResponse extends BaseJsonResponse
     /**
      * {@inheritdoc}
      */
-    public function setData($data = [])// : static
+    public function setData($data = []): JsonResponse|BaseJsonResponse// : static
```

Comment:
Agree, should be static. Comments are still there, because we didn´t add the return type based on those comments, but on the php doc annotation. I will remove them with the next commit.